### DISCUSSION TITLE
请问我怎么实现在umi下面的配置，我真的需要这个配置，其他办法不行

### DISCUSSION THREAD

**Discussion by old-zsh** [umijs/umi#9956] [Q&A]
build: {
    rollupOptions: {
      output: {
        manualChunks: {
          jsonWorker: [`${prefix}/language/json/json.worker`],
          cssWorker: [`${prefix}/language/css/css.worker`],
          htmlWorker: [`${prefix}/language/html/html.worker`],
          tsWorker: [`${prefix}/language/typescript/ts.worker`],
          editorWorker: [`${prefix}/editor/editor.worker`]
        }
      }
    }
  },
下面这种办法，在umi 打包后一直报错，请问怎么实现上面的办法
// chainWebpack: (memo) => {
  //   // 更多配置 https://github.com/Microsoft/monaco-editor-webpack-plugin#options
  //   memo.plugin('monaco-editor-webpack-plugin').use(
  //     new MonacoWebpackPlugin({
  //       languages: ['javascript', 'typescript']
  //     })
  //   );
  //   // return memo;
  // },

**Comment by fz6m** [comment]
简单的配置无法实现多 entry 。请参照 #9863 来配置 monaco 插件解决问题，我是可以正常运行的。

如果运行不起来，提供一个最小复现。
